.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Should not this file be in the `workflows` folder?

Reply:
No, it is [dependabot](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/) configuration, not a workflow.